### ISSUE TITLE
❓ [ QUESTION ]  Should this app be reccomended to those that are not technically inclined?

### ISSUE THREAD

**Issue by Pinaz993** [octoshrimpy/quik#170]
I have a cousin who was using Verizon Messages (AKA Messages+), but it has been [discontinued](https://www.verizon.com/support/verizon-messages-faqs/). One feature she is dearly missing is switch that pops up when you put more than one number in the "To:" box which allows you to choose whether to send the text as a group text or send the same text to multiple recipients. I'd like to recommend this app to her. She's reasonably tech savvy, but not particularly inclined to tinkering. 

#39 indicates that @octoshrimpy would like to polish the app more before attempting to publish on Google Play. I think that's probably wise. However, my cousin, and specifically, her husband, are more than technically inclined enough to install Obtanium if I tell them that it exists. I just don't know how stable the app is, and how likely it is that it will continue to be developed.

Is this app ready to be used by those that do not have programming experience, or should I keep looking?

**Comment by toohazy** [comment]
The app is fairly stable, there are some issues that you can read about in the issues section, but I haven't faced anything that bad.

**Comment by octoshrimpy** [comment]
@Pinaz993: like @toohazy said, the app is pretty stable, and I personally use it as my sms app. If the recomendee runs into any problems, feel free to open up a new issue! (but check it doesn't exist already, please :purple_heart:)

Also, the app is available on f-droid, and the [droidify](https://github.com/Droid-ify/client) app works very well for f-droid repos.

**Comment by octoshrimpy** [comment]
TLDR: 
> Is this app ready to be used by those that do not have programming experience, or should I keep looking?

The app is ready for those that do not have programming experience. latest release will work and require no knowledge of programming to make it work. :)

**Comment by octoshrimpy** [comment]
> [...] switch that pops up when you put more than one number in the "To:" box which allows you to choose whether to send the text as a group text or send the same text to multiple recipients

This sounds like a fantastic feature tho, so feel free to post it as a request if you'd like.

**Comment by Pinaz993** [comment]
> This sounds like a fantastic feature tho, so feel free to post it as a request if you'd like.

I just checked, it's already in the app. It was in QKsms, so that doesn't surprise me.
Thanks for being so responsive. I'll be recommending it to her today.
